index.html

```diff
@@ -49,129 +54,268 @@ <h4>I’m a Software Developer</h4>
-                        <div class="row-skills">
-                            <ul>
-                                <li>CSS</li>
-                                <li>HTML</li>
-                                <li>Bootstrap</li>
-                                <li>Ruby</li>
-                            </ul>
+                        <div class="card-container-row">
+                            <div class="img-desktop">
+                                <a href="#" class="image-bg"><img src="images/Img Placeholder (1).png" alt="portfolio" class="hidden"></a>
+                                <a href="#"><img src="images/Img Placeholder (1).png" alt="project portfolio" class="mobile" width="350px"></a>
+                            </div>
+                            <div class="text-container-row">
+                                <h2>Multi-Post Stories</h2>
+                                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                                <div class="row-skills">
+                                    <ul>
+                                        <li>CSS</li>
+                                        <li>HTML</li>
+                                        <li>Bootstrap</li>
+                                        <li>Ruby</li>
+                                    </ul>
+                                </div>
+                                <button class="btn-container" type="submit">See project</button>
+                            </div>
                         </div>
-                        <button type="submit">See project</button>
                     </div>
                 </div>
 
-                <div class="cards-container">
-                    <div class="card-container">
-                        <h3 class="title-card">Profesional Art Printing Data</h3>
-                        <div class="desc">
-                            <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                <!-- SECTION DESKTOP FOR PROJECTS GRID START HERE -->
+
+                <div class="project-container hidden">
+                    <div class="projects-1 active">
+                        <div class="cards-container-hidden">
+                            <div class="card-container">
+                                <h3 class="title-card">Profesional Art Printing Data</h3>
+                                <div class="desc">
+                                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.</p>
+                                </div>
+                                <div class="row-card-skills">
```

Comment:
- Good job trying implementing some of the changes requested by the previous reviewer :clap: , kindly make sure to align your stacks as requested to move to the next project :recycle: 
![Screenshot from 2023-05-05 15-42-21](https://user-images.githubusercontent.com/97834160/236475594-061c4018-ef24-4ef9-b018-8af6e0d568f4.png)